src/com/ka/spreadsheet/diff/SpreadSheetDiffer.java

```diff
@@ -100,7 +112,8 @@ public static int doDiff(SpreadSheetDiffCallback diffCallback) throws Exception
       diffCallback.reportMacroOnlyIn(hasMacro1);
     }
 
-    diffCallback.reportWorkbooksDiffer(isDiff, WORKBOOK1, WORKBOOK2);
+    diffCallback.reportWorkbooksDiffer(isDiff);
+	diffCallback.finish();
```

Comment:
indentation needs fixing

Reply:
Done.  Fixed tabs everywhere in all src/ and test/com files (all due to my changes) to 2 spaces.